models/articles-model.js

```diff
@@ -12,9 +12,32 @@ exports.selectArticle = (article_id) => {
 };
 
 exports.selectArticles = () => {
-  return db.query(`SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at,
+  return db
+    .query(
+      `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at,
    articles.votes, articles.article_img_url, COUNT (comments.article_id) AS comment_count FROM articles JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id 
-   ORDER BY articles.created_at DESC;`).then(({ rows }) => {
-     return rows;
-   })
-}
+   ORDER BY articles.created_at DESC;`
+    )
+    .then(({ rows }) => {
+      return rows;
+    });
+};
+
+exports.selectComments = (article_id) => {
+  return db
+    .query(
+      `SELECT comments.* FROM comments JOIN articles ON comments.article_id = articles.article_id 
```

Comment:
It isn't really necessary to use a join unless we want to access a field that isn't in the comments table